test/e2e/util/util.go

```diff
@@ -134,14 +134,21 @@ func WaitForGuestClient(t *testing.T, ctx context.Context, client crclient.Clien
 	return guestClient
 }
 
-func WaitForNReadyNodes(t *testing.T, ctx context.Context, client crclient.Client, n int32) []corev1.Node {
+func WaitForNReadyNodes(t *testing.T, ctx context.Context, client crclient.Client, n int32, hc *hyperv1.HostedCluster) []corev1.Node {
```

Comment:
Can we just pass the platform here and not the entire hostedcluster?

Reply:
Yes modified.